noxfile.py

```diff
@@ -25,57 +26,64 @@
 CARTOPY_CACHE_DIR = os.environ.get("HOME") / Path(".local/share/cartopy")
 
 
-def venv_cached(session):
-    """
-    Determine whether the nox session environment has been cached.
-
-    Parameters
-    ----------
-    session: object
-        A `nox.sessions.Session` object.
+def session_lockfile(session: nox.sessions.Session) -> Path:
+    """Return the path of the session lockfile."""
+    return Path(
+        f"requirements/ci/nox.lock/py{session.python.replace('.', '')}-linux-64.lock"
+    )
```

Comment:
```suggestion
    file_name = f"py{session.python.replace('.', '')}-linux-64.lock"
    file_path = Path("requirements") / "ci" / "nox.lock" / file_name
    return file_path
```

Reply:
I get the concept, but this is three lines instead of one, and is it more readable?  As far as I can tell, `pathlib.Path` understands / within a string as an OS agnostic path separator

Reply:
>As far as I can tell, `pathlib.Path` understands / within a string as an OS agnostic path separator

I didn't know that - this was my main reason for suggesting. Regarding readability: FWIW I do find the 3 lines more readable but I'm not concerned either way.